src/app/meshes/views/MeshDetailTabsView.vue

```diff
@@ -18,8 +18,11 @@
       </template>
 
       <NavTabs
-        class="route-mesh-view-tabs"
-        :tabs="tabs"
+        :children="route.children"
+        :active="route.active"
+        i18n-prefix="meshes.routes.item.navigation"
+        :filter-predicate="(route) => route.name !== 'external-service-list-view'"
```

Comment:
If you reuse `DataCollection` here, you don't need to reimplement the same functionality inside `NavTabs` (the presentational component).

Reply:
> What if we don't have tabs depending on what application we are in?

I’m not aware of a case where this occurs currently. Currently, the degree of flexibility we need is deciding what has tabs or not.

> haven't locked this functionality up inside of NavTabs.

Since this isn’t currently something that’s conditional, I don’t see how adding DataCollection is any better than simply removing NavTabs where we don’t need them.